### ISSUE TITLE
stop creating "coremods" folder

### ISSUE THREAD

**Issue by UpcraftLP** [MultiMC/MultiMC5#3348]
When creating and launching a new instance in MultiMC, there will be a `coremods` folder inside.
This folder is created for ANY instance, even vanilla ones (or for other mod loaders like Fabric).

This folder hasn't been needed by Forge [ever since 1.6.1](https://forums.minecraftforge.net/topic/9998-coremods-folder-adding-coremods-treecapitator), and now, 7 years later, it's just unused bloat.

Would it be possible to simply not create it for instances that are newer than 1.5.2 (or 1.6.1 if you really care about those few alpha builds of forge)?

**Comment by Zetabite** [comment]
AFAIK that would need a whole new system to be implemented on waht is stored in the version info are stored. MMC doesnt diffrentiate between pre and post 1.6.1, but some other versions. There are more important stuff to fix than just not creating an empty folder. It is a few bytes at best, and since one barely looks into the instance folder, isnt a big issua visually wise

**Comment by UpcraftLP** [comment]
>  since one barely looks into the instance folder

Users probably don't, but when I am working on modpacks I do very often. Sure there are more pressing matters, but still worth having this issue imo, so that it can be implemented at some point in the future.

**Comment by Zetabite** [comment]
Yeah, just wanted to let you know that this is probably very low prio on any list

**Comment by HassaFelipe** [comment]
> > since one barely looks into the instance folder
> 
> Users probably don't, but when I am working on modpacks I do very often. Sure there are more pressing matters, but still worth having this issue imo, so that it can be implemented at some point in the future.

You can set core/API/lib mods as occult and then configure the folder for don't display occult files.

This way, you don't have to worry about them anymore and they continued to work.

**Comment by HassaFelipe** [comment]
> When creating and launching a new instance in MultiMC, there will be a `coremods` folder inside.
> This folder is created for ANY instance, even vanilla ones (or for other mod loaders like Fabric).
> 
> This folder hasn't been needed by Forge [ever since 1.6.1](https://forums.minecraftforge.net/topic/9998-coremods-folder-adding-coremods-treecapitator), and now, 7 years later, it's just unused bloat.
> 
> Would it be possible to simply not create it for instances that are newer than 1.5.2 (or 1.6.1 if you really care about those few alpha builds of forge)?

Sorry I went through this, but you can still do it with the "coremods" folder but you will need to copy the instances to keep this configuration.

**Comment by UpcraftLP** [comment]
> You can set core/API/lib mods as occult and then configure the folder for don't display occult files.
> 
> This way, you don't have to worry about them anymore and they continued to work.

WHat do you mean by "occult"? Do you mean hidden folders? Yes, that is one option, but each OS handles that differently (UNIX systems often require folder names to start with a dot, which would of course no longer be the same name)

Also, hiding these folders isn't the issue, imo they should not be created in the first place, since they serve no purpose in modern Minecraft installations (see my original comment).
